apps/ima/behaviours/reset-human-rights-summary.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+module.exports = superclass => class extends superclass {
+  saveValues(req, res, next) {
+    if(req.sessionModel.get('family-members') !== undefined) {
+      if(req.sessionModel.get('human-rights-claim') === 'no' &&
+      JSON.stringify(req.sessionModel.get('family-members').aggregatedValues.length) > 0) {
+        req.sessionModel.set('family-members').aggregatedValues = [];
```

Comment:
Rather than setting the ` req.sessionModel.set('family-members').aggregatedValues` to` []`  could it be better to unset it  i.e. ` req.sessionModel.unset('family-members')` if the answer is changed to 'no', as `family-members` is not set if 'no' is selected in the first instance?

Reply:
I've changed it to now use the unset function instead of setting the array as blank.